doc/squashfs-mount.1

```diff
@@ -0,0 +1,32 @@
+.TH SQUASHFS-MOUNT "1" "July 2024" "squashfs-mount 1.0.0" "User Commands"
```

Comment:
One comment here: Maybe we could drop the 1.0.0 version number behind the tool name, or make it get the version from the source tree. Otherwise we might need to bump the version manually each time. Does that make sense?

Reply:
And please feel free to adapt it as you see it fit. It was just a very quick made up manpage so that the tool has one installed on the system (as ideally each installed utility should be accompanied with a manpage)

Reply:
Indeed, I've added a bash script to set the version (tag) and fill in the variables `@DATE@`, `@VERSION@` in the manpage.